apps/fyler/src/fyler_pool.erl

```diff
@@ -331,6 +337,11 @@ next_task(#state{max_active = Max, tasks = Tasks, buf_size = BufSize,
     {error, eexist} -> ok
   end,
 
+  case Active of
+    [] -> fyler_monitor:start_monitor();
+    _ -> ok
+  end,
+
   NewTask = Task#task{file = File#file{dir = TmpDir, tmp_path = filename:join(Dir, Path)}},
```

Comment:
Line 345 is too long:   NewTask = Task#task{file = File#file{dir = TmpDir, tmp_path = filename:join(Dir, Path)}},.